addons/medical_gui/stringtable.xml

```diff
@@ -1074,5 +1074,26 @@
             <German>Partiell %1</German>
             <Turkish>Parsiyel %1</Turkish>
         </Key>
+        <Key ID="STR_ACE_Medical_GUI_IGUI_Category">
+            <English>ACE Medical</English>
+            <Russian>ACE: медицина</Russian>
```

Comment:
This colon was removed and the `m` capitalized in that recent russian translation PR.

Reply:
Suggest the change then.